Arduino_Sub_DAQ/SiMlx.ino

```diff
@@ -0,0 +1,75 @@
+/*************************************************** 
+  This is a library example for the MLX90614 Temp Sensor
+
+  Designed specifically to work with the MLX90614 sensors in the
+  adafruit shop
+  ----> https://www.adafruit.com/products/1748
+  ----> https://www.adafruit.com/products/1749
+
+  These sensors use I2C to communicate, 2 pins are required to  
+  interface
+  Adafruit invests time and resources providing this open source code, 
+  please support Adafruit and open-source hardware by purchasing 
+  products from Adafruit!
+
+  Written by Limor Fried/Ladyada for Adafruit Industries.  
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <Wire.h>
+#include <Adafruit_MLX90614.h>
+#define IR1 0x5C
+#define IR2 0x5A
+#define IR3 0x5B
+#define IR4 0x5D
+Adafruit_MLX90614 mlx;
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  Serial.println("Adafruit MLX90614 test");  
+
+  if(!mlx.begin()) {
+    
+    Serial.println("Error connecting to MLX sensors. Check wiring.");
+    while (1);
+      } 
+ 
+  //Serial.print("Emissivity = "); Serial.println(mlx.readEmissivity());
+  Serial.println("================================================");
+  
+}
+
+
+void loop() {
+  
+  mlx.AddrSet(IR1); 
+  Serial.print("IR1: ");
+  Serial.print("Ambient = "); Serial.print(mlx.readAmbientTempF()); 
+  Serial.print("*F\tObject = "); Serial.print(mlx.readObjectTempF()); Serial.println("*F");
+  mlx.temp1 = mlx.readObjectTempF();
```

Comment:
`readObjectTempF()` is already being called on the preceding line. Instead of performing a read twice back-to-back, consider assigning it to `mlx.temp1` the first time

Reply:
My understanding based on messing with the code is that the second time readObjectTempF() is called on, it's for calculating compute_average_temp. Removing this function directly affects that calculation by setting any mlx.temp used to 0, so compute_average_temp becomes less than what it should be.

Reply:
It's the same data, but two separate reads. I am not asking you to remove the function, only to use it once:
```
mlx.temp1 = mlx.readObjectTempF();
Serial.print("*F\tObject = "); Serial.print(mlx.temp1); Serial.println("*F");
```

Read it into `tempx`, then print, and you can re-use it for computing the average.